docs/USER_GUIDE.md

```diff
@@ -104,6 +104,15 @@ without any arguments:
 Since `krew` itself is a plugin also managed through `krew`, running the upgrade
 command may also upgrade your `krew` version.
 
+### Krew upgrade check
```

Comment:
This is too high up.

Let's move this below to the main user journey and call it `## Disabling update checks`